docs/1.getting-started/10.deployment.md

```diff
@@ -149,3 +149,17 @@ Nuxt 3 can be deployed to several cloud providers with a minimal amount of confi
 - :icon{name="simple-icons:render" class="h-5 w-4 inline mb-2"} [Render](https://nitro.unjs.io/deploy/providers/render)
 - :icon{name="ph:cloud-duotone" class="h-5 w-4 inline mb-2"} [Stormkit](https://nitro.unjs.io/deploy/providers/stormkit)
 - :icon{name="simple-icons:vercel" class="h-5 w-4 inline mb-2 text-black dark:text-white"} [Vercel](https://nitro.unjs.io/deploy/providers/vercel)
+
+## Cloudflare Proxy
```

Comment:
I think we could make it more _CDN Proxy_ section to open for more future advice about CDN deployments. Also other CDNs have similar features that need to be disabled.